src/lib/pipeline.ml

```diff
@@ -165,13 +165,33 @@ module Full (Bfx: Extended_edsl.Semantics) = struct
         in
         sname, Stdlib.fastq_of_input i)
 
+  (** 
+     A record to join a value and its saved version.
+
+     Previous versions of Eidisco had a [Bfx.save] construct of type
+     ['a repr -> 'a repr] which would “hide” saved values, we would
+     then put [Bfx.save ~name] calls everywhere and forget about
+     them. The new [save] from Biokepi is explicit, so to avoid too
+     much rewriting of this pipeline we pack values together.
+
+  *)
+  type 'a saved = {
+    content: 'a Bfx.repr;
```

Comment:
should we just call this `repr` instead of `content`? Asking this because of the following:

```ocaml
  (* ... *)
  | None, Some s, _ -> Some (Bfx.hlarp (`Seq2hla s.content))
  | None, None, Some s -> Some (Bfx.hlarp (`Optitype s.content))
  (* ... *)
```

versus the previous revision:

```ocaml
   (* ... *)
   | None, None, Some s -> Some (Bfx.hlarp (`Optitype s))
   (* ... *)
```
to me, `content` is a completely new term to keep track of so would rather have something re-used here (but this is me being picky, so feel free to ignore)